virttest/remote_commander/remote_interface.py

```diff
@@ -128,6 +129,32 @@ def __setstate__(self, state):
         self.msg = state[1]
 
 
+class CmdQuery(object):
+    """Command-msg-request from VM to avocado-vt test.
+    """
+    def __init__(self, *args, **kargs):
+        """
+        Command for asking from VM to avocado-vt.
+
+        :param args: Something pickable. Is irrelevant for messagenger.
+        :param kargs: Something pickable. Is irrelevant for messagenger.
```

Comment:
Typo? s/messagenger/messenger
And the description statement here could be more accurate.